packages/docs/themes/nimatron/source/css/_samples.scss

```diff
@@ -243,3 +220,114 @@ $spacing-units: (
   background-image: url('data:image/svg+xml,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Cpath%20d%3D%22M13.5%2010.5V13C13.5%2013.3%2013.3%2013.5%2013%2013.5H3C2.7%2013.5%202.5%2013.3%202.5%2013V3C2.5%202.7%202.7%202.5%203%202.5H5.5%22%20stroke%3D%22%231662DD%22%20stroke-miterlimit%3D%2210%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%2F%3E%0A%3Cpath%20d%3D%22M8%208L14%202%22%20stroke%3D%22%231662DD%22%20stroke-miterlimit%3D%2210%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%2F%3E%0A%3Cpath%20d%3D%22M9.5%201.5H14.5%22%20stroke%3D%22%231662DD%22%20stroke-miterlimit%3D%2210%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%2F%3E%0A%3Cpath%20d%3D%22M14.5%206.5V1.5%22%20stroke%3D%22%231662DD%22%20stroke-miterlimit%3D%2210%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%2F%3E%0A%3C%2Fsvg%3E%0A');
   vertical-align: middle;
 }
+
+// Design Token Examples
+
+.sample-token {
+  display: block;
+  width: 21px;
+  height: 7px;
+  margin: $spacing-xs auto 0;
+  border-radius: 4px;
+}
+
+// Token Examples: Color
+.sample-token--color-primary-light {
+  background: $color-primary-light;
+}
+
+.sample-token--color-primary-base {
+  background: $color-primary-base;
+}
+
+.sample-token--color-primary-dark {
+  background: $color-primary-dark;
+}
+
+.sample-token--color-danger-light {
+  background: $color-danger-light;
+}
+
+.sample-token--color-danger-base {
+  background: $color-danger-base;
+}
+
+.sample-token--color-danger-dark {
+  background: $color-danger-dark;
+}
+
+.sample-token--color-primary-bg {
+  background: $color-primary-base;
+}
+
+// Token Examples: Background
+.sample-token--color-primary-bg-disabled {
+  background: $color-primary-light;
+}
+
+.sample-token--color-danger-bg {
+  background: $color-danger-base;
+}
+
+.sample-token--color-danger-bg-disabled {
+  background: $color-danger-light;
+}
+
+// Token Examples: Outline
+.sample-token--color-primary-outline {
+  background: $focus-ring-primary;
+}
+
+.sample-token--color-danger-outline {
+  background: $focus-ring-danger;
+}
+
+// Token Examples: Shadow
+.sample-token--box-shadow-default {
+  box-shadow: $box-shadow-default;
+}
+
+.sample-token--text-body {
+  color: $text-body;
+}
+
+.sample-token--text-heading {
+  color: $text-heading;
+}
+
+.sample-token--text-sub {
+  color: $text-sub;
+}
+
+.sample-token--text-danger {
+  color: $text-danger;
+}
+
+.type-body-sample--table {
+  width: 90%;
+  white-space: nowrap;
+
+  .type-sample:nth-of-type(1) .type-sample--example {
+    font-size: $size-body-l;
+  }
+
+  .type-sample:nth-of-type(2) .type-sample--example {
+    font-size: $size-body-m;
+  }
+
+  .type-sample:nth-of-type(3) .type-sample--example {
+    font-size: $size-body-s;
+  }
```

Comment:
I realize this is sample css, but is this additional selector specificity required for some reason?

We should strive to keep all selectors as flat as possible to help make our lives easier in the future

Reply:
I agree, this was a temporary addition to the css to test something out. removing!